vuepress/docs/next/docs/reference/entando-cli.md

```diff
@@ -236,107 +228,178 @@ An `env.json` file to configure the application URLs and client credentials must
 
 Instructions to export a bundle, including how to configure `env.json`, can be found in the [Export and Publish a Bundle tutorial](../../tutorials/create/pb/export-bundle-from-application.md).
 
-## Profile Management
-To manage and switch between different ent configurations, use `ent profile`. 
 
-To switch between different Entando Applications, even if they are in different clusters, `ent profile` uses Kubernetes contexts, kubeconfig files, and/or custom commands (refer to `ent profile first-use-readme`).
+### Custom Resource Wrapper
```

Comment:
ECR actually references Entando Component Repository - although some of the new options do have more to do with custom resources. Maybe change line 233 "to facilitate custom resource operations" -> "for interaction with the Entando Component Repository". The section title should probably change too.

Side note, the ECR is likely to change to Local Hub in 7.1 but this API likely won't change so we'll need to watch for that.